### ISSUE TITLE
Img Plugin can not load base64 encoded images.

### ISSUE THREAD

**Issue by nitch193** [rdkcentral/Lightning-SDK#305]
This happend when I as trying to render a simple base64 encoded image on my app, It says on the console Error: texture load error Image load error

**Comment by michielvandergeest** [comment]
Hi @nitch193 could you please provide a code snippet? That makes it easier for us to understand what you're doing exactly and allows us to easily replicate. Thanks!

**Comment by nitch193** [comment]
Hi @michielvandergeest  thanks for replying. Actually in our app we fetch the data from our api and one of the endpoint of our api returns base64 encoded image string, and when i a trying to render the image in our app it shows the error as mentioned above. Here is the code snippet: 
` Image: {mount: 0.5, x: 480, y:270, w: 300, h: 300, src: this.data.url}`
```here this.data.url``` is a base64 encoded image string.

**Comment by michielvandergeest** [comment]
Hi @nitch193, I think that in your case you can skip the `Img` plugin all together, especially since you're using the original method on it.

Instead you could do something like:

```js
Poster : {
  src: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAA...',
  w: 250,
  h: 250
}
```

**Comment by nitch193** [comment]
Hey @michielvandergeest , actually I tried what you mentioned above but by using this method i will lose all the methods that Img plugin provides like .orignal(), .cover(), . contain () and etc.

**Comment by michielvandergeest** [comment]
@nitch193 we've added a fix for this in PR #319. The fix has already been merged to `dev` and will be part of the next Lightning-SDK release.

**Comment by nitch193** [comment]
When will be the next release?

**Comment by sandeep-vedam** [comment]
Hey @nitch193 ,

This issue is fixed at #319 and is available in release 5.0.0 of SDK
